### ISSUE TITLE
centos7-plbase errors due to inclusion of python3 with centos7 now

### ISSUE THREAD

**Issue by coatless** [PrairieLearn/PrairieLearn#1728]
Look at a recent PR #1726 

On rebuilding the docker image, we have: 

```bash
ln: failed to create symbolic link '/usr/bin/python3': File exists
```

From Slack, @trombonekenny mentioned:

> CentOS 7 recently updated and included python3

Need to remove:

https://github.com/PrairieLearn/PrairieLearn/blob/3fdd44e858eeff4eaa21068e1cb1d1fe84516264/environments/centos7-plbase/Dockerfile#L19-L21

**Comment by coatless** [comment]
Maybe also remove: 

https://github.com/PrairieLearn/PrairieLearn/blob/3fdd44e858eeff4eaa21068e1cb1d1fe84516264/environments/centos7-plbase/Dockerfile#L51

**Comment by trombonekenny** [comment]
FWIW, I'm having this problem on my CentOS 7 desktop too. I'm not sure yet how we (as a community) solve this. We could try ditching the python36u packages and see if the system-installed python3 does enough?

**Comment by trombonekenny** [comment]
I'm playing with this in PR #1719

**Comment by trombonekenny** [comment]
Resolved with #1719
